Matlab/color_Mett.m

```diff
@@ -0,0 +1,185 @@
+function output = color_Mett(map, minColor, verbosity)
+% This function is based on Blake Roberts Mills code for R, to create colormaps based on different paintings from the Metropolitan Museum of Art.
+% You can check the project at https://github.com/BlakeRMills/MetBrewer/tree/main
+% Inputs:
+%   map: the name of the map you want to use.
+%   minColor: the minimum color you want to use, if the selected map does not have enough colors, they will be interpolated.
+% Outputs:
+%   output: the colormap.
+% Example:
+%   output = color_Mett('Monet', 21);
+% validColorMaps = ["Austria", "Cassatt1","Cassatt2","Cross","Degas","Derain","Egypt","Gauguin", "Greek",...
+%     "Hiroshige","Hokusai1","Hokusai2","Hokusai3","Homer1", "Homer2", "Ingres", "Isfahan1", "Isfahan2", ...
+%     "Juarez", "Klimt","Lakota","Manet", "Monet", "Moreau", "Morgenstern", "Nattier","Navajo", "NewKingdom","Nizami"...
+%     "OKeffe1","OKeffe2","Peru1","Peru2","Pillement", "Pissaro", "Redon", "Renoir", "Robert", "Signac" "Stevens", "Tara",...
+%     "Tsimshian","Thomas", "Tiepolo", "Troy", "VanGogh1", "VanGogh2", "VanGogh3","Veronese","Wissing"];
+% MGG 13.12.21
+% 18.10.22 Added missing color palettes
+
+if nargin<2; minColor = 1; end
+if nargin<3; verbosity = false; end
+    
+    % Parse input
+    validColorMaps = ["Austria", "Cassatt1","Cassatt2","Cross","Degas","Derain","Egypt","Gauguin", "Greek",...
+    "Hiroshige","Hokusai1","Hokusai2","Hokusai3","Homer1", "Homer2", "Ingres", "Isfahan1", "Isfahan2", ...
+    "Juarez", "Klimt","Lakota","Manet", "Monet", "Moreau", "Morgenstern", "Nattier","Navajo", "NewKingdom","Nizami"...
+    "OKeffe1","OKeffe2","Peru1","Peru2","Pillement", "Pissaro", "Redon", "Renoir", "Robert", "Signac" "Stevens", "Tara",...
+    "Tsimshian","Thomas", "Tiepolo", "Troy", "VanGogh1", "VanGogh2", "VanGogh3","Veronese","Wissing"];
+    map= validatestring(map,validColorMaps);
+    if verbosity; fprintf(['color_Met>> Retrieving colormap ', upper(char(map)), '... \n']);end
+    
+    % Anonymous function to convert from HEX to RGB, credits to Chad Greene
+    % See hex2rgb.
+    rgb = @(hex)(reshape(sscanf(strrep(hex, '#','').','%2x'),3,[]).'/255);
+    
+    % This color palet was originally created for R by Blake Robert Mills.
+    % See https://github.com/BlakeRMills/MetBrewer/tree/main
+   function output = color_Mett(map, minColor, verbosity)
```

Comment:
You have broken your script by accident, defining the function and documentation again, starting here in line 35

Reply:
I didnot had time the other day to check it, but seems like the commit got corrupted. Should be fixed now.